react-ui/src/components/LogInForm.js

```diff
@@ -34,73 +34,93 @@ class logInForm extends Component {
     });
   };
 
-
   login = event => {
     let email = this.state.loginParams.email;
     let password = this.state.loginParams.password;
-    var params = {email: email, password: password}
-    axios.get(`/public/login/?${queryString.stringify(params)}`)
-    .then(res => {
-      if(res.data!==""){
-        this.setState({
-          email : res.data.email,
-          userType: res.data.userType,
-          firstName: res.data.firstName,
-          personId: res.data.workerId || res.data.studentId,
-          accessToken: res.data.accessToken,
-          islogged: true
+    let validinput = true;
+    var params = { email: email, password: password }
+
+    if (email === "" && password === "") {
+      validinput = false
+      alert("Please enter email and password")
+      event.preventDefault();
+    } else if (email === "") {
+      validinput = false;
+      alert("Please enter an email")
+      event.preventDefault();
+    } else if (password === "") {
+      validinput = false;
+      alert("please enter an password")
```

Comment:
Hi Kyle,
For consistency, I would recommend also capitalizing the P in please for this alert as well.

Reply:
wording has been updated